utils/datafilegen/prjxraydbconverter

```diff
@@ -14,6 +15,18 @@ class PrjxrayDbConverter:
     def __init__(self, src):
         # Using abspath to avoid symlink resolving
         self.src = Path(os.path.abspath(src))
+        #Read parts.yaml file and get path
+        with open(self.src.parent/"mapping/parts.yaml", 'r') as f:  
+            data = yaml.safe_load(f)
+            if data.get(self.src.name):
+                device = data.get(self.src.name).get("device")
```

Comment:
Let's just check the key presence in `if` instead of returning the object/None:
```
if self.src.name in data:
```

Reply:
Fixed as suggested.